### ISSUE TITLE
Create simple cluster fails when getting kubectl config

### ISSUE THREAD

**Issue by sandcastle** [kubernetes/kops#1303]
I'm having issues with creating a simple cluster, it seems to be falling over when trying to get the kubectl config.

As you will see in the logs below it looks like its missing the `=` when specifying the output

_Doesnt work from terminal:_

```
kubectl config view --output json

Error: flag needs an argument: --output
Run 'kubectl help' for usage.
```

_Works from terminal:_

```
kubectl config view --output=json
kubectl config view -o json
```

## Versions

kops version: 1.4.4
kubernetes-cli vesion: 1.5.1_1

## Logs

```
I1229 11:18:39.503858   43540 populate_cluster_spec.go:196] Defaulting DNS zone to: XYZ
I1229 11:18:42.007567   43540 executor.go:68] Tasks: 0 done / 53 total; 26 can run
I1229 11:18:44.385605   43540 executor.go:68] Tasks: 26 done / 53 total; 11 can run
I1229 11:18:45.012903   43540 executor.go:68] Tasks: 37 done / 53 total; 14 can run
I1229 11:18:45.898520   43540 executor.go:68] Tasks: 51 done / 53 total; 2 can run
I1229 11:18:46.021921   43540 executor.go:68] Tasks: 53 done / 53 total; 0 can run
I1229 11:18:46.124212   43540 kubectl.go:68] error running kubectl config view --output json:
I1229 11:18:46.124244   43540 kubectl.go:69] Error: flag needs an argument: --output
Run 'kubectl help' for usage.
W1229 11:18:46.124266   43540 update_cluster.go:139] error reading kubecfg: error getting config from kubectl: error running kubectl
I1229 11:18:46.311516   43540 update_cluster.go:150] Exporting kubecfg for cluster
Wrote config for cloud.something.xyz to "/Users/glenn/.kube/config"
```

**Comment by justinsb** [comment]
Thanks for the report.  I haven't yet been able to reproduce the problem (`kubectl config view --output json` works fine for me).  Is this on OSX?  Is this the kubectl from brew?  Are you maybe using an unusual shell?

**Comment by sandcastle** [comment]
I'm using: macOS, brew & zsh as shell

**Comment by justinsb** [comment]
Figured it out!  You have an old version of kubectl somewhere on your PATH.  I'm actually thinking kubectl 1.0.  I downloaded a few different versions, and it looks like the `--output json` syntax was added in 1.1

We could fix this, but I'm inclined to say that having a version of kubectl that old is likely to cause more problems, and so it's good to have it around.

If you `which kubectl` I think that will be the kubectl that is on your PATH.  (And I think `kubectl version` will show you are not using the version 1.5 from brew)

**Comment by chrislovecnm** [comment]
Any update?

**Comment by solitaryr** [comment]
tldr; I ran into this same issue with my linux desktop.  justinsb's comment fixed my issue.

If you've ever installed the Google Cloud CLI, it may contain kubectl.  When I checked my path, it was first so the version I added to /usr/local/bin (linux) was not getting called.  Updating the CLI removed this.  I had a _really_ old copy of GCloud CLI.
